components/automate-ui/src/app/components/create-object-modal/create-object-modal.component.ts

```diff
@@ -33,6 +41,23 @@ export class CreateObjectModalComponent implements OnInit {
     });
   }
 
+  ngOnChanges(changes: SimpleChanges): void {
+    // if a new list of projects to populate dropdown with is passed in we update the dropdown
+    const checked = false;
+    if (hasIn('assignableProjects.currentValue', changes)) {
+      changes.assignableProjects.currentValue.forEach((proj: Project) =>
+        this.projects[proj.id] = { ...proj, checked });
```

Comment:
I think there is a bug here: this assumes the project list will only grow. But someone might delete a project, too. So need to empty out `this.projects` first before the loop.